docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -77,17 +94,44 @@ const finalTheme = createTheme({
 
 {{"demo": "GlobalThemeOverrideCallback.js"}}
 
-### Using `sx` (experimental) syntax
+### Using the `sx` experimental syntax
```

Comment:
This is nitpicking at this point but I think this formatting makes the most sense here.

```suggestion
### The `sx` syntax (experimental)
```

Reply:
I believe _syntax_ makes sense in this context as it is regarding its usage within the theme object, not as an actual component prop. 🤔

Reply:
Add more info here. Technically, it is not a prop (React props) but a function. Maybe it is better to say "shorthand syntax" or "sx function"?

Reply:
I think syntax is accurate because it is pointing out how you can use `pr: 1` instead of `paddingRight: 1`.